src/main/java/nextstep/courses/domain/session/entity/SessionEntity.java

```diff
@@ -106,9 +117,23 @@ public String toString() {
             ", price=" + price +
             ", payType='" + payType + '\'' +
             ", maxStudentCount=" + maxStudentCount +
-            ", coverImage='" + coverImage + '\'' +
+            ", coverImageId='" + coverImageId + '\'' +
             ", startDateTime=" + startDateTime +
             ", endDateTime=" + endDateTime +
             '}';
     }
+
+    public Session toSession(SessionCoverImage sessionCoverImage) {
```

Comment:
수강생 목록까지 인자로 받는다면 수강생을 포함한 객체로 만들 수 있지 않을까요?
```suggestion
    public Session toSession(SessionCoverImage sessionCoverImage, List<NsUser> students) {
```